javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,93 @@
+//Fullname 
+function getFullname(firstname, surname, useFormalName, isMale){
+    let fullname = "";
+    if(!firstname){
+        console.log("please write your firstname");
+        return;
+    }
+    if(!surname){
+        console.log("please write your surname");
+        return;
+    }
+    if (useFormalName) {
+        if (isMale){
+          fullname="Lord ";
+        }
+        else{
+            fullname="Miss ";
+        }
+    }
+    fullname = fullname + firstname + " " + surname;
+    return fullname;
+}
+
+const fullname1= getFullname("Besart","Elezi", true,true);
+console.log(fullname1);
+const fullname2= getFullname("Benjamin","");
+console.log(fullname2);
+
+
+
+//Event application
+function getEventWeekday(daysAfter){
+    const weekDays = ["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"];
+    const today = new Date().getDay();
+    const eventToBeHeld = today + daysAfter;
+    const postionInArray = (eventToBeHeld % 7) - 1;
+    return weekDays[postionInArray];
+}
+console.log(getEventWeekday(9));
```

Comment:
The logic is fine but it doesn't seem to work with the value 1 or 8, for example . Can you spot the bug?

```
> getEventWeekday(1)
undefined
```